modules/virt-creating-filesystem-fusion-access-san.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * virt/fusion_access_SAN/install-configure-fusion-access-san.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-filesystem-fusion-access-san_{context}"]
+= Creating a file system with {FusionSAN}
+
+You need to create a file system to represent your required storage.
+
+The file system is based on the storage available in the worker nodes you selected when creating the storage cluster.
+
+.Prerequisites
+
+* You created a {FusionSAN} storage cluster.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Storage* -> *{FusionSAN}*.
+
+. In the *File systems* tab, click *Create file system*.
+
+. Enter a *Name* for the new file system.
+
+. Select the LUNs that you want to use as the storage volumes for your file system.
+
+. Click *Create file system*. 
++
+The *{FusionSAN}* page reloads, and the new file system appears in the *File systems* tab.
+
+Repeat this procedure for each file system that you want to create.
+
+.Verification
+
+. Watch the *Status* of the file system in the *File systems* tab until it is marked as *Healthy*.
++
+[NOTE]
+====
+This may take several minutes.
+====
+
+. Click on the *StorageClass* for the file system.
+
+. In the *YAML* tab, verify the following:
++
+.. The value in the `name` field is the name of the file system you created.
+.. The value in the `provisioner` field is `spectrumscale.csi.ibm.com`.
+.. The value in the `volBackendFs` field matches the name of the file system you created.
+
++
+.Example YAML
```

Comment:
This block title cannot be assigned to a code block in DITA. In most cases, you can either skip it or replace it with a leading paragraph, the latter will require more complex formatting to work around the substeps.

```suggestion
```